pkg/ttnpb/end_device.go

```diff
@@ -1552,3 +1552,14 @@ func (m *GetEndDeviceRequest) ExtractRequestFields(dst map[string]interface{}) {
 func (m *EndDevice) ExtractRequestFields(dst map[string]interface{}) {
 	m.GetIds().ExtractRequestFields(dst)
 }
+
+// Utility
```

Comment:
Remove this comment and actually comment on the function itself with a description of the functionality.